components/Header.vue

```diff
@@ -34,7 +34,7 @@
         </v-col>
       </v-row>
       <v-row class="nav-row" v-else>
-        <v-col align="left">
+        <v-col align="center">
```

Comment:
I don't think this should be centered, see [designs](https://www.figma.com/file/1QJRKgVI6x69Xi6VkNuPdm/Track-City-Budgets?node-id=0%3A1)